homeassistant/components/zha/core/cluster_handlers/general.py

```diff
@@ -218,6 +218,14 @@ class GreenPowerProxy(ClusterHandler):
 
     BIND: bool = False
 
+    @callback
+    def cluster_command(self, tsn, command_id, args):
+        """Handle commands received to this cluster."""
+        cmd_name = parse_and_log_command(self, tsn, command_id, args)
+
+        if cmd_name == "notification":
```

Comment:
As a note, in the future we may want to use this here: 
`greenpower.GreenPowerProxy.ServerCommandDefs.notification.name`
However, that would depend on the zigpy ZGP PR https://github.com/zigpy/zigpy/pull/1282/. Right now, this PR is independent of that.

(Relevant PR for other zigpy def changes: https://github.com/home-assistant/core/pull/108383)

Reply:
Thanks for your patience as we work thru the issues surrounding the tree that crashed into our house a little over a month ago haha. 

The intentions of this PR were twofold:
- Allow this PR to be reviewed independently of the larger ZGP patch
- Keep the code changes as minimal as possible to allow for expedited review

If you'd like to wait for the main PR to hit first so we can use the name reference as opposed to a magic string I totally get that. No problem on my end.